pkg/cli-runtime/config_pkg_test.go

```diff
@@ -64,12 +64,12 @@ func TestInitKubeConfig_Flag(t *testing.T) {
 	c.Stdout = output
 	c.Stderr = output
 
-	c.KubeConfigFile = "testdata/.kube/config"
+	c.KubeConfigFile = configPath
```

Comment:
nit : not a fan of using a var from other test file. It would be great if its all local to this test file or test case.